.github/workflows/jaseci-studio-test.yml

```diff
@@ -0,0 +1,26 @@
+name: Jaseci Studio Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
@Gim3l Could you update this to only trigger the tests if there are changes in the `jaseci_studio` folder? You can take a look at the jaseci_ai_kit github action workflow file for how to do that.

Reply:
Ok, I made the change.